Resources/ServerInfo/Guidebook/Survival.xml

```diff
@@ -7,14 +7,14 @@ It is generally wise to avoid situations that will cause you harm, because medic
 Your PDA contains both a vessel name and list of crew currently active in your vessel, a reminder for your name if needed, and your assigned job for the shift, allowing you to quickly assess the situation.
 
 ## Emergency Treatment
-In the event of a serious emergency, there's a few things you can do to help ensure your long-term survival, including:
+In the event of a serious emergency, there're a few things you can do to help ensure your long-term survival, including:
```

Comment:
there are (there're doesn't flow well. there's aka there is also was just fine.)